styles.css

```diff
@@ -308,19 +310,15 @@ body {
     cursor: pointer;
     z-index: 10;
     -webkit-transition: .05s linear all;
-        -moz-transform: .05s linear all;
```

Comment:
The `-moz-transform` and `transform` properties should not be deleted since it means that the animation will only appear on webkit based browsers.